test/format/format_config_def.c

```diff
@@ -364,6 +364,10 @@ CONFIG configuration_list[] = {{"assert.read_timestamp", "assert read_timestamp"
   {"transaction.implicit", "implicit, without timestamps, transactions (percentage)", 0, 0, 100,
     100, V_GLOBAL_TRANSACTION_IMPLICIT},
 
+  {"transaction.operation_timeout_ms",
```

Comment:
To capture a discussion with @mm-ng. 
My understand that the timeout applies to the transaction, but the name implies to me it applies to a single operation in the context of the transaction.

Reply:
I'm not sure what the original context was when this option was created, but given that this API is also used by MongoDB I'm a little hesitant to change the name. I can do some digging to see if there is anything on Jira about this.

Reply:
We definitely can't change the name here without a bit of a headache, we'd need to coordinate that with MongoDB.